media_cdn/dualtoken.py

```diff
@@ -15,14 +15,15 @@
 # limitations under the License.
 
 # [START mediacdn_dualtoken_sign_token]
+# [START mediacdn_dualtoken_sign_path_component]
 import base64
 import datetime
 import hashlib
 import hmac
 
 import cryptography.hazmat.primitives.asymmetric.ed25519 as ed25519
 
-
+# [END mediacdn_dualtoken_sign_path_component]
```

Comment:
Can this be right below the import without a space? It will also help solve the lint issue.